### PR TITLE
[original author: mrnikwaws] fixing incorrect stride information on xla tensors

### PR DESCRIPTION
**Pull request by aws-kingrj**
- fixing incorrect xla tensor stride information
- breaks log softmax lowering
- tested on AWS Neuron internal testing
- original author mrnikwaws

### CODE REVIEW

**Comment by JackCaoG** [comment]
Ok I think I know what's going on. The pr from the fork can not use remote cache so build just timeout. Let me see if I can bump the timeout.

**Comment by JackCaoG** [comment]
pr in https://github.com/pytorch/xla/pull/5470. Let me also grant you write access so you can create branch on pytorch/xla directly. This way all of the CI will use the bazel remote cache and runs much faster.

**Comment by JackCaoG** [comment]
@aws-kingrj if you rebase the build should start run pass timeout.

File: torch_xla/csrc/tensor_impl.cpp [hunk 1]
```diff
@@ -161,6 +161,11 @@ at::IntArrayRef XLATensorImpl::strides_custom() const {
   return strides_default();
 }
```
Comment (JackCaoG) [inline torch_xla/csrc/tensor_impl.cpp:163]: You mentioned this `breaks log softmax lowering`. Can you add a test that will fail without this change? The change in general make sense but without a test it is hard to make sure this is correct nor can we prevent this from regressing again.

File: test/cpp/test_aten_xla_tensor_1.cpp [hunk 2]
```diff
@@ -3700,6 +3700,19 @@ TEST_F(AtenXlaTensorTest, TestPowIntExponent) {
   ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestPowFloatScalarBaseIntExponent) {
```
Comment (JackCaoG) [inline test/cpp/test_aten_xla_tensor_1.cpp:3703]: I think the commit is a bit messed up, this seems to be from other pr
